python/hopsworks/core/library_api.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         self._project_id = project_id
         self._project_name = project_name
 
-    def install(self, library_name: str, python_version: str, library_spec: dict):
+    def install(self, library_name: str, name: str, library_spec: dict):
         """Create Python environment for the project"""
```

Comment:
Same about `#Arguments`. It guess it can raise `RestAPIError` as well.

Reply:
This function should actually be hidden, it's only used internally by Environment.install_wheel() etc... But I added the docs string.